src/main/resources/config/application.properties

```diff
@@ -37,17 +37,17 @@ core_case_data.jurisdictionId=${CORE_CASE_DATA_JURISDICTION_ID:SSCS}
 core_case_data.caseTypeId=${CORE_CASE_DATA_CASE_TYPE_ID:Benefit}
 
 # IDAM Java Client
-idam.url=${IDAM_API_URL:http://localhost:5000}
+idam.url=${IDAM_API_URL:http://localhost:5555}
 idam.s2s-auth.totp_secret=${IDAM_S2S_AUTH_TOTP_SECRET:AAAAAAAAAAAAAAAC}
 idam.s2s-auth.microservice=${IDAM_S2S_AUTH_MICROSERVICE:sscs}
 idam.s2s-auth.url=${IDAM_S2S_AUTH:http://localhost:4502}
 idam.oauth2.user.email=${IDAM_SSCS_SYSTEMUPDATE_USER:system.update@hmcts.net}
 idam.oauth2.user.password=${IDAM_SSCS_SYSTEMUPDATE_PASSWORD:Pa55word11}
 idam.oauth2.redirectUrl=${IDAM_OAUTH2_REDIRECT_URL:https://localhost:3000/authenticated}
-auth.idam.client.baseUrl=${IDAM_API_URL:http://localhost:5000}
+auth.idam.client.baseUrl=${IDAM_API_URL:http://localhost:5555}
 auth.provider.service.client.baseUrl=${IDAM_S2S_AUTH:http://localhost:4502}
 auth.idam.client.redirectUri=${IDAM_OAUTH2_REDIRECT_URL:https://localhost:3000/authenticated}
-idam.api.url=${IDAM_API_URL:http://localhost:5000}
+idam.api.url=${IDAM_API_URL:http://localhost:5555}
```

Comment:
Probably shouldn't have committed this as no one else using the stub

Reply:
changes